### PR TITLE
Change date range format which fails to parse in some clients

### PR DESCRIPTION
**Pull request by stevejgordon**
The format used in one of the date range tests fails to parse in some language clients due to the `:` in the date. This PR changes the format to align with that used in other range tests and should avoid the parsing problem.

### CODE REVIEW

**Review by sethmlarson — APPROVED** [review]

**Comment by salvatore-campagna** [comment]
@elasticmachine run Check labels

**Comment by salvatore-campagna** [comment]
@elasticmachine update branch

File: rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/40_range.yml [hunk 1]
```diff
@@ -457,7 +457,9 @@ setup:
               range:
                 field: date
                 ranges:
-                  { from: 2021-05-01T00:00:00Z, to: 2021-05-05T00:00:00Z }
+                 -
+                   from: 2021-05-01T00:00:00Z
+                   to: 2021-05-05T00:00:00Z
```
Comment (nik9000) [inline rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/40_range.yml:462]: I think you can also quote these and they should work in both cases. I know I sort of habitually quote these.

It'd be nice if we could detect cases where this'd break the client in our build or test infrastructure.

Comment (stevejgordon) [reply rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/40_range.yml:462]: I also wondered about that but found an example using this form which I believe works now. I agree, a check to ensure that the client parsers don't break when new tests come in would be nice.

**Review by salvatore-campagna — APPROVED** [review]

**Pull request merged**